### PR TITLE
Add the 'require_last_push_approval' field 

### PR DESCRIPTION
**Pull request by noamd-legit**

### CODE REVIEW

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @noamd-legit !
Could you please also add this field (but as a `*bool`) to the `PullRequestReviewsEnforcementUpdate` struct?

**Comment by noamd-legit** [comment]
@gmlewis Added :)

File: github/repos.go [hunk 1]
```diff
@@ -1064,6 +1066,8 @@ type PullRequestReviewsEnforcementUpdate struct {
 	// RequiredApprovingReviewCount specifies the number of approvals required before the pull request can be merged.
 	// Valid values are 1 - 6 or 0 to not require reviewers.
 	RequiredApprovingReviewCount int `json:"required_approving_review_count"`
+	// RequireLastPushApproval specifies whether the last pusher to a pull request branch can approve it.
+	RequireLastPushApproval *bool `json:"require_last_push_approval"`
```
Comment (gmlewis) [inline github/repos.go:1070]:
```suggestion
	RequireLastPushApproval *bool `json:"require_last_push_approval,omitempty"`
```

**Review by gmlewis — CHANGES_REQUESTED** [review]

**Comment by gmlewis** [comment]
You will probably also need to run `go generate ./...` as described in CONTRIBUTING.md. Sorry for the short reviews... I'm on Android.

**Comment by noamd-legit** [comment]
`go generate` doesn't seem to generate diff after syncing with the main branch

**Comment by gmlewis** [comment]
Did you literally type `go generate ./...` in the top-level directory?
(_Not_ `go generate`.)

**Review by gmlewis — APPROVED** [review]
Thank you, @noamd-legit !
LGTM.
Merging.

**Pull request merged**